juntagrico/util/sessions.py

```diff
@@ -72,8 +79,14 @@ def subscriptions(self):
         :return: dict of subscription_types -> amount, where selected amount > 0
         """
         subscription = self.get('subscriptions', {})
-        sub_types = SubscriptionType.objects.filter(id__in=subscription.keys())
-        return {sub_type: subscription[str(sub_type.id)] for sub_type in sub_types if subscription[str(sub_type.id)]}
+        return get_parts_dict(subscription)
+
+    def extras_enabled(self):
+        return SubscriptionPart.objects.is_extra().exists()
```

Comment:
This always returns to false in my test with the juntagrico test data after adding an extra sub type. The `SubscriptionPart.objects.is_extra()` returns empty.